src/python/TigerGraph_helper/tg_helper.py

```diff
@@ -1,12 +1,9 @@
 import logging, os, pyTigerGraph as tg
 from typing import Optional
-logger = logging.getLogger(__name__)
-logger.setLevel(logging.DEBUG)
-
+logger = logging.getLogger()
```

Comment:
demo ones these should be `logging.getLogger(__name__)` generally
core ones should be util.log.getChild(__name__)

Reply:
what do you mean by "version grade" here?  the move from streamlit 1.21 --> 1.25?

Reply:
yeah -- I reviewed the st version upgrades yesterday, no expected breakages, and sounds like you've manually tested otherwise already, so we're good on that

so just fixing the conflicts + https://github.com/graphistry/graph-app-kit/pull/104#discussion_r1295272818